scvi/model/_destvi.py

```diff
@@ -110,6 +114,10 @@ def from_rna_model(
             trained CondSCVI model
         vamp_prior_p
```

Comment:
same here, either you have vamp_prior_p or vamp_prior_resolution, not both (unless you support for mulitple modes)